### PR TITLE
Update Speech Recognition example (with `learning_rate`)

### PR DESCRIPTION
**Pull request by krshrimali**
## What does this PR do?

Addresses https://github.com/PyTorchLightning/lightning-flash/issues/1112 to add `learning_rate` to the `SpeechRecognition` constructor.

## Before submitting
- [x] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the **[contributor guideline](https://github.com/PyTorchLightning/lightning-flash/tree/master/.github/CONTRIBUTING.md)**, Pull Request section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes?
- [ ] Did you write any **new necessary tests**? [not needed for typos/docs]
- [ ] Did you verify **new and existing tests pass** locally with your changes?
- [ ] If you made a notable change (that affects users), did you **update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)**?

## PR review
 - [x] Is this pull request **ready for review**? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by Borda — APPROVED** [review]

**Review by ethanwharris — APPROVED** [review]
LGTM :smiley:

File: flash_examples/speech_recognition.py [hunk 1]
```diff
@@ -29,16 +29,17 @@
...
 # 3. Create the trainer and finetune the model
 trainer = flash.Trainer(max_epochs=1, gpus=torch.cuda.device_count())
 trainer.finetune(model, datamodule=datamodule, strategy="no_freeze")
 
 # 4. Predict on audio files!
-datamodule = SpeechRecognitionData.from_files(predict_files=["data/timit/example.wav"], batch_size=4)
-predictions = trainer.predict(model, datamodule=datamodule)
-print(predictions)
+predict_datamodule = SpeechRecognitionData.from_files(predict_files=["data/timit/example.wav"], batch_size=4)
+predictions = trainer.predict(model, datamodule=predict_datamodule)
```
Comment (ethanwharris) [inline flash_examples/speech_recognition.py:40]: Let's keep the name datamodule for now just to be consistent with the other examples:
```suggestion
datamodule = SpeechRecognitionData.from_files(predict_files=["data/timit/example.wav"], batch_size=4)
predictions = trainer.predict(model, datamodule=datamodule)
```

File: flash_examples/speech_recognition.py [hunk 2]
```diff
@@ -29,16 +29,17 @@
...
+
+print("Predictions: ", predictions)
```
Comment (ethanwharris) [inline flash_examples/speech_recognition.py:42]: @krshrimali Mind adding a follow-up to print like this in all of the examples?

Comment (krshrimali) [reply flash_examples/speech_recognition.py:42]: Sure, @ethanwharris - will do this tomorrow for all other examples in a separate PR.

**Comment by ethanwharris** [comment]
Closing in favour of #1275

**Pull request closed without merging**